azkaban-migration/src/main/java/azkaban/migration/schedule2trigger/Schedule2Trigger.java

```diff
@@ -210,6 +210,7 @@ private static void file2ScheduleTrigger() throws Exception {
         DateTimeZone timezone = DateTimeZone.forID(timezoneId);
         ReadablePeriod period =
             Utils.parsePeriodString(schedProps.getString("period"));
+        String cronExpression = schedProps.getString("cronExpression");
```

Comment:
As discussed before, azkaban actually retired schedule tables now. We must migrate schedule to trigger when a schedule is created.